tests/ip_api_test.go

```diff
@@ -0,0 +1,200 @@
+package tests
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/phoenixnap/go-sdk-bmc/ipapi"
+	"github.com/stretchr/testify/suite"
+)
+
+type IpApiTestSuite struct {
+	suite.Suite
+	Client *ipapi.APIClient
+	Ctx    context.Context
+}
+
+func (suite *IpApiTestSuite) SetupSuite() {
+	// Set configuration
+	configuration := ipapi.NewConfiguration()
+	configuration.Host = "127.0.0.1:1080"
+	configuration.Scheme = "http"
+
+	// Set the context background
+	suite.Ctx = context.WithValue(context.Background(), "accessToken", "ACCESSTOKENSTRING")
+	suite.Ctx = context.WithValue(suite.Ctx, "serverIndex", nil)
+
+	// New ApiClient
+	suite.Client = ipapi.NewAPIClient(configuration)
+}
+
+func (suite *IpApiTestSuite) TearDownTest() {
+	TestUtilsImpl{}.resetExpectations()
+}
+
+func (suite *IpApiTestSuite) VerifyCalledOnce(expectationId string) {
+	// Result retrieved from server's verification
+	// Verifying expectation matched exactly once.
+	verifyResult := TestUtilsImpl{}.verifyExpectationMatchedTimes(expectationId, 1)
+
+	// Asserts a successful result
+	suite.Equal(202, verifyResult.StatusCode)
+}
+
+func (suite *IpApiTestSuite) TestGetIpBlocks() {
+	// Generate payload
+	request, response := TestUtilsImpl{}.generatePayloadsFrom("ipapi/ip_blocks_get", "./payloads")
+
+	// Extract the response expectation id
+	expectationId := TestUtilsImpl{}.setupExpectation(request, response, 1)
+
+	// Operation Execution
+	result, _, _ := suite.Client.IPBlocksApi.IpBlocksGet(suite.Ctx).Execute()
```

Comment:
why not handle the other two returns simply by logging them

Reply:
because we don't need to log them

Reply:
why not its only gonna help out when it comes having tests failing for whatever reason i think